recipes/gateway.rb

```diff
@@ -39,21 +39,24 @@
       false
     end
 
-  password = node['cdap']['cdap_site']['router.ssl.keystore.password']
-  keypass =
-    if node['cdap']['cdap_site'].key?('router.ssl.keystore.keypassword')
-      node['cdap']['cdap_site']['router.ssl.keystore.keypassword']
-    else
-      node['cdap']['cdap_site']['router.ssl.keystore.password']
-    end
-  path = node['cdap']['cdap_site']['router.ssl.keystore.path']
-  common_name = node['cdap']['security']['ssl_common_name']
-  jks =
-    if node['cdap']['cdap_site'].key?('router.ssl.keystore.type') && node['cdap']['cdap_site']['router.ssl.keystore.type'] != 'JKS'
-      false
-    else
-      true
-    end
+  if ssl_enabled.to_s == 'true'
```

Comment:
Yes and no. This was done on purpose to prevent us from writing out defaults to cdap-security.xml in the case where security isn't enabled. I mentioned that in the description. Should I add it as a comment, instead?